omnia.yml

```diff
@@ -21,10 +21,10 @@
     - cluster_validation
 
 - name: Gather facts from all the nodes
-  hosts: all
+  hosts: manager, compute, login_node, nfs_node
```

Comment:
If user is opting for powervault me4 support, then in that case powervault device IP will also be present in the inventory and gathering facts will fail for powervault IP since for omnia template the credentials used are omnia credentials and powervault credentials are different. That is the reason we are specifying group names to ignore powervault IP if it is present.